CONTRIBUTING.md

```diff
@@ -161,6 +161,21 @@ Cluster API maintains the most recent release/releases for all supported API and
 
 - Exceptions can be filed with maintainers and taken into consideration on a case-by-case basis.
 
+### Removal of v1alpha3 & v1alpha4 apiVersions
+
+The last releases using v1alpha3 (v0.3.x) and v1alpha4 (v0.4.x) apiVersions have been both EOL since ~ April 2022.
+The apiVersions have been kept around to allow upgrades from v0.3.x and v0.4.x Cluster API versions to v1.x. Given that 
+those releases have been EOL since a while now, we are going to remove the apiVersions in upcoming releases:
+* v1.5:
+  * Kubernetes API server will stop serving the v1alpha3 apiVersion
+* v1.6:
+  * v1alpha3 apiVersion will be removed from the CRDs
+  * Kubernetes API server will stop serving the v1alpha4 apiVersion
+* v1.7
+  * v1alpha4 apiVersion will be removed from the CRDs
+
+For more details and latest information please see the following issue: [Removing v1alpha3 & v1alpha4 apiVersions](https://github.com/kubernetes-sigs/cluster-api/issues/8038).
```

Comment:
```suggestion
The last releases supporting v1alpha3 (v0.3.x) and v1alpha4 (v0.4.x) apiVersions have been both EOL since April 2022.
The apiVersions have been kept around to allow upgrades from v0.3.x and v0.4.x Cluster API versions to v1.x. Given that these releases have been EOL for a significant period, we are going to remove the apiVersions in upcoming releases:
* v1.5:
  * Kubernetes API server will stop serving the v1alpha3 apiVersion
* v1.6:
  * v1alpha3 apiVersion will be removed from the CRDs
  * Kubernetes API server will stop serving the v1alpha4 apiVersion
* v1.7
  * v1alpha4 apiVersion will be removed from the CRDs

For more details and latest information please see the following issue: [Removing v1alpha3 & v1alpha4 apiVersions](https://github.com/kubernetes-sigs/cluster-api/issues/8038).
```

Reply:
Thx, done